client/.babelrc

```diff
@@ -1,8 +1,11 @@
 {
     "presets": ["@babel/env", "@babel/preset-react"],
+          "plugins":
+          ["@babel/plugin-proposal-class-properties"],
```

Comment:
Since we are no longer using class properties in the ResultsPage component, let's not add this plugin.